examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -529,20 +532,23 @@ class WriteAttributesResponse : public ModelCommandResponse
 
     bool HandleResponse(uint8_t * message, uint16_t messageLen) const override
     {
+        bool success = true;
         ChipLogProgress(chipTool, "WriteAttributesResponse (0x04):");
+
         // struct writeAttributeResponseRecord[]
         while (messageLen)
         {
             CHECK_MESSAGE_LENGTH(1);
-            if (!CheckStatus(chip::Encoding::Read8(message))) // zclStatus
+            uint8_t status = chip::Encoding::Read8(message); // zclStatus
+            success        = CheckStatus(status);
+            if (status != 0)
             {
-                return false;
+                CHECK_MESSAGE_LENGTH(2);
+                ChipLogProgress(chipTool, "  %s: 0x%04x", "attributeId", chip::Encoding::LittleEndian::Read16(message)); // attribId
```

Comment:
So this is a failed write, correct?  Should the log message be clearer about that?

Reply:
Yes. The type of error will be printed by `CheckStatus`.

Also chip-tool will see that as an error and it will be printed when chip-tool returns.